weblate/lang/models.py

```diff
@@ -523,7 +523,7 @@ class Plural(models.Model):
         verbose_name=ugettext_lazy('Plural definition source'),
         choices=(
             (SOURCE_DEFAULT, ugettext_lazy('Default plural')),
-            (SOURCE_GETTEXT, ugettext_lazy('Gettext plural formula')),
+            (SOURCE_GETTEXT, ugettext_lazy('Plural gettext formula')),
```

Comment:
It's plural formula coming from gettext, so I believe the original string was correct.

Reply:
In my head that is a "Plural gettext formula", I could be wrong.